.github/workflows/project-automations.yml

```diff
@@ -106,12 +106,12 @@ jobs:
     if: github.event_name == 'pull_request' && github.event.action == 'opened'
     steps:
       - name: Move release PR to ${{ env.in_review }}
-        if: startsWith(github.head_ref, 'merge/prerelease/minor-into-prerelease/major') || startsWith(github.head_ref, 'merge/master-into-prerelease/minor') || startsWith(github.head_ref, 'dependabot/**')
+        if: startsWith(github.head_ref, 'merge/**') || startsWith(github.head_ref, 'dependabot/**')
         uses: leonsteinhaeuser/project-beta-automations@v2.0.1
         with:
           gh_token: ${{ env.gh_project_token }}
           organization: ${{ env.gh_organization }}
           project_id: ${{ env.gh_project_id }}
           resource_node_id: ${{ github.event.pull_request.node_id }}
           status_value: ${{ env.in_review }}
-          custom_field_values: '[{\"name\": \"Priority\",\"type\": \"single_select\",\"value\": \"🌋 Urgent\"}, {\"name\": \"Sprint\",\"type\": \"iteration\",\"value\": \"@current\"}]'
+          custom_field_values: '[{\"name\": \"Priority\",\"type\": \"single_select\",\"value\": \"🌋 Urgent\"},{\"name\": \"Sprint\",\"type\": \"iteration\",\"value\": \"@current\"}]'
```

Comment:
im assuming no space is fine?

Reply:
based on the [action documentation](https://github.com/leonsteinhaeuser/project-beta-automations#json-definition) I believe so